Rule-003Tests.postman_collection.json

```diff
@@ -52,19 +52,18 @@
"cfg\": \"029@1.0\",\r\n                                \"rules\": [\r\n                                    {\r\n                                        \"id\": \"003@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    },\r\n                                    {\r\n                                        \"id\": \"005@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    }\r\n                                ]\r\n                            }\r\n                        ]\r\n                    },\r\n                    {\r\n                        \"id\": \"002@1.0\",\r\n                        \"host\": \"http://openfaas:8080\",\r\n                        \"cfg\": \"1.0\",\r\n                        \"typologies\": [\r\n                            {\r\n                                \"id\": \"030@1.0\",\r\n                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",\r\n                                \"cfg\": \"030@1.0\",\r\n                                \"rules\": [\r\n                                    {\r\n                                        \"id\": \"003@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    },\r\n                                    {\r\n                                        \"id\": \"006@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    }\r\n                                ]\r\n                            },\r\n                            {\r\n                                \"id\": \"031@1.0\",\r\n                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",\r\n                                \"cfg\": \"031@1.0\",\r\n                                \"rules\": [\r\n                                    {\r\n                                        \"id\": \"003@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    },\r\n                                    {\r\n                                        \"id\": \"007@1.0\",\r\n                                        \"host\": \"http://openfaas:8080\",\r\n                                        \"cfg\": \"1.0\"\r\n                                    }\r\n                                ]\r\n                            }\r\n                        ]\r\n                    }\r\n                ]\r\n            }\r\n        ]\r\n    }\r\n}",
```

Comment:
please replace https://frmfaas.sybrin.com with http://gateway.openfaas:8080 (even in the network sub-map) for ALL future tests..........................